compact/ecal/bic/bic.xml

```diff
@@ -39,6 +39,15 @@
     <constant name="EcalBarrel_Calorimeter_offset"
       value="(EcalBarrel_Calorimeter_zmax - EcalBarrel_Calorimeter_zmin)/2.0"/>
 
+    <constant name="EcalBarrel_LightGuide_length"      value="5*cm"/>
+    <constant name="EcalBarrel_EndOfSectorBox_length"  value="12*cm"/>
+    <constant name="EcalBarrel_LightGuide_zmin"        value="EcalBarrel_Calorimeter_zmin + EcalBarrel_LightGuide_length"/>
+    <constant name="EcalBarrel_LightGuide_zmax"        value="EcalBarrel_Calorimeter_zmax + EcalBarrel_LightGuide_length"/>
+    <constant name="EcalBarrel_EndOfSectorBox_zmin"    value="EcalBarrel_Calorimeter_zmin + EcalBarrel_EndOfSectorBox_length"/>
```

Comment:
Now that we have `EcalBarrel_EndOfSectorBox_zmin`  we  should substitute <constant name="EcalBarrel_Readout_zmin"          value="273.75*cm"/>